mmv1/templates/terraform/examples/app_engine_flexible_app_version.tf.erb

```diff
@@ -17,10 +17,22 @@ resource "google_project_service" "service" {
   disable_dependent_services = false
 }
 
+resource "google_service_account" "custom_service_account" {
+  project      = google_project_iam_member.gae_api.project
```

Comment:
This is currently causing terraform to crash:

```
Error: Cycle: google_project_iam_member.gae_api, google_service_account.custom_service_account
```

```suggestion
  project      = google_project_service.service.project
```

Reply:
ah lol